src/lib.rs

```diff
@@ -22,6 +24,8 @@ use types::{TorrentAddArgs, TorrentAdded};
 pub struct TransClient {
     url: String,
     auth: Option<BasicAuth>,
+    session_id: Option<String>,
+    client: Client,
```

Comment:
Is that really necessary to expose the client? Please try to avoid it

Reply:
Aren't they private by default in Rust?